pkg/build/builder/cmd/scmauth/password.go

```diff
@@ -77,7 +79,11 @@ func (u UsernamePassword) Setup(baseDir string, context SCMAuthContext) error {
 		}
 		defer gitconfig.Close()
 
-		fmt.Fprintf(gitconfig, UserPassGitConfig, gitcredentials.Name())
+		configContent := fmt.Sprintf(UserPassGitConfig, gitcredentials.Name())
+
+		glog.V(5).Infof("Adding username/password credentials to git config:\n%s\n", configContent)
```

Comment:
is this going to potentially dump git credentials to the build log?  if so we can't do that.

Reply:
No, it will output this:
```
I1023 12:41:32.441781      11 password.go:84] Adding username/password credentials to git config:
# credential git config
[credential]
   helper = store --file=/tmp/gitcredentials.196096979
```

Reply:
ok, then it lgtm if you're ready for it to be merged.

Reply:
yup, will remove the wip shortly, I think I may need to remove the other one you commented on below though because you could include a secret in a gitconfig fragment that you provide.